ehsm_kms_service/ehsm_kms_server.js

```diff
@@ -1,13 +1,13 @@
 const express = require('express')
-const ffi = require('ffi-napi');
 const crypto = require('crypto')
 const app = express();
 const logger = require('./logs/logger');
+const ehsm_napi = require('./ehsm_napi');
 const { ehsm_kms_params, ehsm_keyspec_t, ehsm_keyorigin_t} = require('./ehsm_kms_params.js')
 
 app.use(express.json());
 
-const port = 3000;
+const port = 9000;
```

Comment:
i wondering that, if the port 9000 is in using, what will be happed? ehsm-kms can't start?